kitsune/messages/forms.py

```diff
@@ -1,20 +1,20 @@
 from django import forms
 from django.utils.translation import gettext_lazy as _lazy
 
-from kitsune.sumo.form_fields import MultiUsernameField
-
-
-TO_PLACEHOLDER = _lazy("username1, username2,...")
+from kitsune.sumo.form_fields import MultiUsernameOrGroupnameField
 
 
 class MessageForm(forms.Form):
     """Form send a private message."""
 
-    to = MultiUsernameField(
+    to = MultiUsernameOrGroupnameField(
         label=_lazy("To:"),
-        widget=forms.TextInput(
-            attrs={"placeholder": TO_PLACEHOLDER, "class": "user-autocomplete"}
```

Comment:
It would be nice to have a placeholder. You can pass in the form `__init__` the request.user and update dynamically the text based on whether they can select groups.

Reply:
The placeholder wasn't functional in production in the first place! But, because I aim to please, I have patched `jquery.tokeninput.js` and the form `__init__` to get the thing back on the rails.